tests/test_bfh.py

```diff
@@ -325,29 +356,106 @@ class Consistent(Mapping):
 
     def test_empty_fields_serialize_as_none_valid_or_no(self):
         """We aren't making assumptions here. Call validate if you want it."""
-        class FirstSchema(Schema):
-            wow = IntegerField(required=True)
-            umm = IntegerField(required=False)
+        first_schema, other_schema, my_map = set_up_schemas()
+        source = first_schema(wow=1)
```

Comment:
Since this is new functionality on the Field class, I think that putting all these new tests in the TestMappings class is kind of a misplacement. Unittest on as low a level as possible, and add higher level integration tests sparingly, as necessary.